api/models.py

```diff
@@ -185,3 +185,29 @@ class SecurityUser(User):
 
     class Meta:
         verbose_name = "Security User"
+
+
+class AssetLog(models.Model):
+    """Stores checkin/Checkout asset logs"""
+    CHECKIN = "Checkin"
+    CHECKOUT = "Checkout"
+
+    option = (
+        (CHECKIN, "Checkin"),
+        (CHECKOUT, "Checkout"),
+    )
+    item = models.ForeignKey(Item,
```

Comment:
(sorry i should have added this to the initial request on the ticket but) can we make this reference the `serial_number` and not the item id? ( see [here](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ForeignKey.to_field))
note: for that to happen https://www.pivotaltracker.com/story/show/156196172 should be completed on this PR

Reply:
alright. will do that

Reply:
@nelmandela this isn't done yet. "reference the item serial_number and not the item id"

Reply:
we should have `to_field='serial_number',` here

Reply:
thanks for that. on it